src/hotspot/share/opto/idealGraphPrinter.cpp

```diff
@@ -377,9 +377,6 @@ void IdealGraphPrinter::visit_node(Node *n, bool edges, VectorSet* temp_set) {
     const Type *t = node->bottom_type();
     print_prop("type", t->msg());
     print_prop("idx", node->_idx);
-#ifdef ASSERT
-    print_prop("debug_idx", node->_debug_idx);
-#endif
```

Comment:
Why you removed this?

Reply:
print_prop() only works for int.  I could add an overload that works for uint64_t, but then I realized debug_idx is redundant for IGV, as we already have the compile_id and node _idx.

Reply:
Removing `debug_idx` from IGV graph dumps makes sense to me as well.